ext/dom/tests/gh14698.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+GH-14698 crash on DOM node dereference
+--EXTENSIONS--
+dom
+--CREDITS--
+YuanchengJiang
+--FILE--
+<?php
+  $dom = new DOMDocument;
+  $dom->loadHTML('<span title=""y">x</span><span title=""z">x</span>');
```

Comment:
So there's a syntax error in the HTML code, i.e. the double "". This causes the warnings we see in the EXPECTF section. You don't need that to repro the issue. Fixing that makes the test a little cleaner. Also please don't indent top-level code.

```suggestion
  $dom->loadHTML('<span title="y">x</span><span title="z">x</span>');
```